next.config.js

```diff
@@ -0,0 +1,73 @@
+/** @type {import('next').NextConfig} */
+const path = require('path')
+
+const csp =
+  process.env.NODE_ENV === 'development'
+    ? ''
+    : "default-src 'self'; img-src 'self'; media-src 'self'; script-src 'self' 'unsafe-eval' 'unsafe-inline';"
+
+const nextConfig = {
+  distDir: `dist/${process.env.NODE_ENV}`,
+  reactStrictMode: true,
+  experimental: {
+    serverActions: { bodySizeLimit: '25mb' }
+  },
+  devIndicators: {
+    buildActivityPosition: 'bottom-right'
+  },
+  sassOptions: {
+    includePaths: [path.join(__dirname, 'styles')]
+  },
+  poweredByHeader: false,
+  async headers() {
+    return [
+      {
+        source: '/(.*)',
+        headers: [
+          {
+            key: 'X-Frame-Options',
+            value: 'DENY'
+          },
+          {
+            key: 'X-Content-Type-Options',
+            value: 'nosniff'
+          },
+          {
+            key: 'Permissions-Policy',
+            value:
+              'microphone=(), geolocation=(), usb=(), payment=(), autoplay=(), display-capture=(), encrypted-media=(), gamepad=(), gyroscope=(), midi=(), picture-in-picture=(), publickey-credentials-get=(), screen-wake-lock=()'
```

Comment:
Had to check whether this is allow or deny, from mdn "() (empty allowlist): The feature is disabled in top-level and nested browsing contexts. The equivalent for <iframe> allow attributes is 'none'.". Might be good to document, first I assumed this was a whitelist.

Reply:
TODO: add documentation

Reply:
Added documentation.